src/hotspot/share/jvmci/vmStructs_jvmci.cpp

```diff
@@ -411,17 +410,16 @@
   declare_constant(JVM_ACC_HAS_FINALIZER)                                 \
   declare_constant(JVM_ACC_IS_CLONEABLE_FAST)                             \
   declare_constant(JVM_ACC_IS_HIDDEN_CLASS)                               \
-  declare_constant(JVM_ACC_FIELD_INTERNAL)                                \
-  declare_constant(JVM_ACC_FIELD_STABLE)                                  \
-  declare_constant(JVM_ACC_FIELD_HAS_GENERIC_SIGNATURE)                   \
   declare_constant(JVM_ACC_IS_VALUE_BASED_CLASS)                          \
+  declare_constant(FieldInfo::FieldFlags::_ff_injected)                   \
+  declare_constant(FieldInfo::FieldFlags::_ff_stable)                     \
+  declare_constant(FieldInfo::FieldFlags::_ff_generic)                    \
```

Comment:
I don't think `_ff_generic` is used in the JVMCI Java code so this entry can be deleted. Please double check the other entries.

Reply:
_ff_generic removed.
_ff_stable is used in `HotSpotResolvedJavaFieldImpl.isStable()`.
_ff_injected is used in `HotSpotResolvedJavaFieldImpl.isInternal()`